pkg/ui/app/containers/events.tsx

```diff
@@ -50,6 +74,7 @@ export function getEventInfo(e: Event) {
   return {
     timestamp: TimestampToMoment(e.timestamp).fromNow(),
```

Comment:
this isn't a timestamp, it's a duration. The field is named incorrectly.  The "sortableTimestamp" field, on the other hand, actually is a timestamp.